examples/python/gui_demo.py

```diff
@@ -220,6 +220,9 @@ def tree_widget_create(self, parent_frame):
         scroll_bar.pack(fill=tk.Y, side=tk.RIGHT)
 
         parent_frame.add(frame)
+        tree.tag_configure('ok', foreground='olive drab')
+        tree.tag_configure('warning', foreground='orange')
+        tree.tag_configure('error', foreground='red')
```

Comment:
You should define the colour palette at a single place and refer to those colours by a variable name. You are hardcoding "olive drab" several times in this single MR. Once the code starts growing and you'll want to change the colour, you'll be struggling to find all the zillion of places where some exotic colour has been specified, and there's a high probability that you'll miss some places. Or someone will want to use dark mode, the colour will be hardly readable on dark background, and it will be extremely painful to change it.